app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    count_friends = None

```

Comment:
The `count_friends` attribute is initialized to `None`. If this is intended to be a counter, it might be better to initialize it to `0`.